src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/legacy/controller/LegacyPeriodLengthController.kt

```diff
@@ -29,6 +33,33 @@ class LegacyPeriodLengthController(
   private val legacyPeriodLengthService: LegacyPeriodLengthService,
   private val eventService: PeriodLengthDomainEventService,
 ) {
+
+  @PostMapping
+  @ResponseStatus(HttpStatus.CREATED)
+  @Operation(
+    summary = "Create a new period length of a single sentence",
+    description = "Creates a new period length of a single sentence",
```

Comment:
minor - but can we change this swagger doc to be something like
Creates one or more period length records for a sentence.